### DISCUSSION TITLE
Need Quick Shop/View on PLP pages

### DISCUSSION THREAD

**Discussion by rutulritz** [breezefront/community#47] [3. Integrations]
Hi there, 

Can you please let me know or guide to implement the Quick shop or Quick view instead of add to cart on product category pages.

Or any extension completible with Breeze theme, please let me know.

Thanks

**Comment by vovayatsyuk** [comment]
We have an [AjaxPro](https://swissuplabs.com/ajax-pro-extension-for-magento-2.html) module that adds a QuickView button.

Here how it looks like in [ArgentoForce (Breeze) theme](https://argentotheme.com/):

https://user-images.githubusercontent.com/306080/212270062-b1f50f34-312c-42ef-927d-a1e76eb3c03e.mp4

**Reply by rutulritz** [reply]
Thanks @vovayatsyuk .

**Reply by parthvisavadiya** [reply]
What if we have any product slider on pdp and we opens quickview on pdp? Have you face product image sticks on pdp even after close the quickview popup?
